From 60c008c33579a1423f910c9d8f98e5e739b5c304 Mon Sep 17 00:00:00 2001
From: Tao Huang <huangtao@rock-chips.com>
Date: Mon, 14 Dec 2020 18:30:20 +0800
Subject: [PATCH] input: sensors: compass: Remove useless config
 COMPASS_MMC328X

No rule to make target 'drivers/input/sensors/compass/mmc328x.c',
needed by 'drivers/input/sensors/compass/mmc328x.o'.

Signed-off-by: Tao Huang <huangtao@rock-chips.com>
Change-Id: Ic29c4ebfb85f482470e7e5d24f2e6429187fda37
---
 drivers/input/sensors/compass/Kconfig  | 10 ----------
 drivers/input/sensors/compass/Makefile |  1 -
 2 files changed, 11 deletions(-)

diff --git a/drivers/input/sensors/compass/Kconfig b/drivers/input/sensors/compass/Kconfig
index cdb6287fe92b..91a19ffe23ea 100644
--- a/drivers/input/sensors/compass/Kconfig
+++ b/drivers/input/sensors/compass/Kconfig
@@ -54,15 +54,5 @@ config COMPASS_AK09918
   To compile this driver as a module, choose M here: the module
   will be called AK09911.
 
-config COMPASS_MMC328X
-  tristate "Mmc328x 3-Axis Magnetometer"
-  depends on I2C
-  help
-  Say yes here to build support for mmc3280 3-Axis
-  Magnetometer.
-
-  To compile this driver as a module, choose M here: the module
-  will be called mmc3280. 
-
 endif
 
diff --git a/drivers/input/sensors/compass/Makefile b/drivers/input/sensors/compass/Makefile
index 064976c7f508..f42c2aa2dd17 100644
--- a/drivers/input/sensors/compass/Makefile
+++ b/drivers/input/sensors/compass/Makefile
@@ -6,4 +6,3 @@ obj-$(CONFIG_COMPASS_AK8975)	+= ak8975.o
 obj-$(CONFIG_COMPASS_AK8963)	+= ak8963.o
 obj-$(CONFIG_COMPASS_AK09911)	+= ak09911.o
 obj-$(CONFIG_COMPASS_AK09918) 	+= ak09918.o
-obj-$(CONFIG_COMPASS_MMC328X)	+= mmc328x.o 
-- 
2.35.3

